.drone.yml

```diff
@@ -74,7 +74,7 @@ steps:
     image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
     environment:
       #DOCKER_PASSWORD:
-       # from_secret: DOCKER_PASSWORD
+       #from_secret: DOCKER_PASSWORD
```

Comment:
@DivineTaminangHO Please delete anything in comments which is not being used by pipeline